pinakes/main/approval/tests/unit/test_templates.py

```diff
@@ -19,13 +19,9 @@ def test_duplicate_template_title(self):
 
         tenant = TenantFactory()
         title = "fred"
-        template = TemplateFactory(tenant=tenant, title=title)
-        with pytest.raises(IntegrityError) as excinfo:
+        TemplateFactory(tenant=tenant, title=title)
```

Comment:
Because the returned value of a factory is not used, in my opinion it's better to call `create` method explicitly here.

Reply:
@cutwater These tests are checking if a constraint validation is raised when we try to add an object into the database with the same parameters. So its the same call made twice the second one triggers the exception. So I am not sure if we should have 2 different calls one that calls the Factory and another one that calls the Model.objects.create, shouldn't they both be the same. If a factory is meant to ease creation of objects by providing defaults majority of the tests use the Factory method instead of creating the objects directly.

Reply:
1. This particular comment is referring to factory usage. In my opinion `Factory()` call floating in the air is confusing. You could use `TemplateFactory.create(tenant=tenant, title=title)` instead of calling it, which does the same, but is more explicit on action, even if the result of action is ignored.
2. I'd say the creation, that is being tested is better to be implemented via model, so that we're testing model itself and not it's wrapper.